.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,44 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ main ]
```

Comment:
You need to change this and the one below on line 8 to your default branch, which is `master` for now.